src/tests/api-tests.ts

```diff
@@ -2906,6 +2906,20 @@ describe('api tests', () => {
     expect(fetchBlockByHeight.status).toBe(200);
     expect(fetchBlockByHeight.type).toBe('application/json');
     expect(JSON.parse(fetchBlockByHeight.text)).toEqual(expectedResp);
+
+    const fetchBlockByBurnBlockHeight = await supertest(api.server).get(
+      `/extended/v1/block/by_burn_block_height/${block.burn_block_height}`
+    );
+    expect(fetchBlockByBurnBlockHeight.status).toBe(200);
+    expect(fetchBlockByBurnBlockHeight.type).toBe('application/json');
+    expect(JSON.parse(fetchBlockByBurnBlockHeight.text)).toEqual(expectedResp);
+
+    const fetchBlockByBurnBlockHash = await supertest(api.server).get(
+      `/extended/v1/block/by_burn_block_hash/${block.burn_block_hash}`
+    );
+    expect(fetchBlockByBurnBlockHash.status).toBe(200);
+    expect(fetchBlockByBurnBlockHash.type).toBe('application/json');
+    expect(JSON.parse(fetchBlockByBurnBlockHash.text)).toEqual(expectedResp);
```

Comment:
Can you add a couple simple tests for the burn height and burn hash 404 code paths?